pvlib/irradiance.py

```diff
@@ -154,8 +154,11 @@ def to_doy(x): return x                                 # noqa: E306
 
 def aoi_projection(surface_tilt, surface_azimuth, solar_zenith, solar_azimuth):
     """
-    Calculates the dot product of the solar vector and the surface
-    normal.
+    Calculates the dot product of the sun position unit vector and the surface
+    normal unit vector; in other words, the cosine of the angle of incidence.
```

Comment:
to improve this even slightly more, can you use LaTeX to add the equation?

    $$AOI_{projection} = \\cos{ \\beta } \\cos{ \\left( \\text{ solar zenith } \\right) } + \\sin{ \\beta } \\sin{ \\left( \\text{ solar zenith } \\right) } \\cos{ \\left( \\text{ solar azimuth} \\right) - \\left( \\text{ surface azimuth} \\right) }$$

might require some creative thinking about symbols, and some text here or elsewhere to define them. Just an idea